django/thunderstore/api/cyberstorm/views/community_list.py

```diff
@@ -7,14 +8,17 @@
 
 
 class CommunityPaginator(PageNumberPagination):
+    max_page_size = 100
     page_size = 100
+    page_size_query_param = "page_size"
```

Comment:
Would strongly prefer fixed page sizes to reduce cache complexity, and I don't think we have a need to use non-standard one?

Reply:
Yeah I think we can live without it for now. I just added the support here since it was present on the frontend before I started working on it. Only place where it would be needed in addition to the community list would have been the landing page where a small batch top communities are shown, but I'm not sure if that would be handled by this endpoint anyway. (And the frontend implementation didn't even use pagination, but sliced the result set on client 🙄)